src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null
```

Comment:
you render this component only if user exists, so you shouldn't handle null value here